bundle/edu.gemini.lchquery.servlet/src/main/scala/edu/gemini/lchquery/servlet/LchQueryParam.scala

```diff
@@ -94,7 +94,14 @@ object LchQueryParam {
         Option(inv).map(i => s"${i.getFirst} ${i.getLast}").filterNot(_.trim.isEmpty)
 
       val spProg = toSPProg
-      (investigatorName(spProg.getGsaPhase1Data.getPi) :: spProg.getGsaPhase1Data.getCois.asScala.map(investigatorName).toList).
+      val piName = for {
+        f <- Option(spProg.getPIFirstName)
+        l <- Option(spProg.getPILastName)
+        n = s"$f $l".trim
+        if !n.isEmpty
+      } yield n
```

Comment:
Huh... weird about Travis. I had to rebuild the project, I think, to get things to work properly. Perhaps it has something to do with this since the version number on the lch bundle is different now?